tests/components/sensor/test_rmvtransport.py

```diff
@@ -117,12 +120,22 @@ def tearDown(self):
         """Stop everything that was started."""
         self.hass.stop()
 
+    @asyncio.coroutine
     @patch('RMVtransport.RMVtransport.get_departures',
```

Comment:
Patching a coroutine doesn't work. Also standard unit tests doesn't support coroutine test methods. Probably use pytest fixtures while moving to separate pytest test functions.

Reply:
Thanks, I'll rewrite the tests.